apps/web/src/data/en/index.ts

```diff
@@ -925,6 +925,8 @@ export const loggedInData = {
           previewMode: 'Preview',
           previewIsActiveHint: 'Edit the exercise again',
           previewIsDeactiveHint: 'Preview the exercise',
+          dummyAnswers: 'Dummy answers',
+          addDummyAnswer: 'Add a dummy answer',
```

Comment:
String from the design are "Falsche Antwortmöglichkeiten" and "falsche Antwort hinzufügen" respectively. If this English translation is not optimal, please suggest a better one 🙏